security/bug-bounty.md

```diff
@@ -6,6 +6,8 @@ Coordinated disclosure rules
 * Alert us as soon as possible. Let us know upon discovery of a potential security issue, and we’ll make every effort to quickly correct the issue.
 * Let us triage, discuss, then publish. Provide us with a reasonable amount of time to fix the issue, and discuss with us before publishing it elsewhere.
 * Please use demo accounts. Not following the account creation steps below will render your report ineligible for a bounty. This includes automated/scripted account creation.
+* Using automated scanners and testing tools to send large number of requests (more than 1000 per day per endpoint) against the endpoints in the products, without prior notice or alerting to security@clever.com could render your report ineligible for bounty and could inevitably prompt us to disallow you from our program. If you are reporting a vulnerability regarding rate limiting or denial of service, you should let us know about the email address, account name, account ID, or other identifier associated with your testing setup, so that we can filter and attribute those large number of requests to you.
```

Comment:
1000 seems very small. But at the same time what we are asking them to do is to notify us before doing it.

I am wondering if we should ask them to add a HTTP header like https://hackerone.com/datastax?type=team would that be more helpful

Reply:
~1000 was based on the time /signup endpoint was blasted. The jist from reading this section should be that if the researchers let us know, this limit does not apply.